composer/trainer/devices/device_cpu.py

```diff
@@ -28,6 +30,11 @@ def precision_context(self, precision: Union[str, Precision]) -> Generator[None,
         precision = Precision(precision)
         if precision == Precision.FP32:
             yield
+        elif precision == Precision.BF16:
```

Comment:
Consider not enabling this yet, as looks like bf16 for CPU is not fully baked in pytorch just yet: https://github.com/pytorch/pytorch/issues/55374